lib/session.js

```diff
@@ -117,26 +117,28 @@ function autoStoreData (req, res, next) {
   next()
 }
 
-function getSessionNameFromServiceName (serviceName) {
-  return 'govuk-prototype-kit-' + (Buffer.from(serviceName, 'utf8')).toString('hex')
+function getSessionNameFromWorkingDirectory (workingDirectory) {
+  return 'govuk-prototype-kit-' + (Buffer.from(workingDirectory, 'utf8')).toString('hex')
 }
 
 function getSessionMiddleware () {
   const config = getConfig()
+  const workingDirectory = process.cwd()
 
   // Session uses service name to avoid clashes with other prototypes
```

Comment:
This comment needs to refer to working directory instead of service name.

Reply:
good spot - fixed now!